### DISCUSSION TITLE
MVP Trust Flavors / Categories

### DISCUSSION THREAD

**Discussion by tayler6000** [eigentrustnetwork/ETN#18] [Ideas]
If anyone has ideas for some MVP trust flavors, please put them here.

**Comment by tayler6000** [comment]
The current flavor I was thinking of is "AI Safety Research" and would apply to the Rob Miles and similar discords.

**Reply by plexish** [reply]
Yup, this will be in the initial set. I think each flavor should come with a description, this one could be:

**AGI Safety Research**
> Assign this flavor to nodes (people or orgs) which have produced research which makes it more likely that humanity will solve the AI control problem, resulting in an aligned superintelligence.

Others in the AI safety cluster include:

**AGI Safety Content Recommendations**
> Assign this flavor to URLs of content which you think is most valuable for people to read when learning about AGI Safety.

Defined as `one node out from the AGI Safety graph`. Should have a largeish amount (~200) pointed towards null by default, so that people who trust more content have higher weight.

**AGI Safety Ecosystem Development**
> Assign this flavor to nodes (people or orgs) which have provided value to you as a researcher or other researchers by improving the ecosystem (outreach, training, support, conferences, coaching, accommodation, etc).

This is defined as either: `one node out from the AGI Safety Research graph` OR preferably `First apply the AGI Safety Research graph, then run a second ET calculation with those added as trusted nodes to your set of directly trusted nodes`. Both of these compositions are good to have as available options.

**AGI Safety**
> Assign this flavor to nodes (people or orgs) who you trust on AGI Safety.

This one should include a person's trust points added to either of the previous two graphs in the calculation. Votes on Rob's Discord should be directly into this category, unless a person explicitly imports them into a different flavor.

**AGI Safety Content Recommendations**
> Assign this flavor to URLs of content which you think is most valuable for people to read when learning about AGI Safety.

Defined as `one node out from the AGI Safety graph`.

**Reply by tayler6000** [reply]
> This is defined as either: `one node out from the AGI Safety Research graph` OR preferably `First apply the AGI Safety Research graph, then run a second ET calculation with those added as trusted nodes to your set of directly trusted nodes`. Both of these compositions are good to have as available options.

Can you expand on this a little bit? And if it's different also touch on a secondary flavor?

**Reply by plexish** [reply]
> > This is defined as either: `one node out from the AGI Safety Research graph` OR preferably `First apply the AGI Safety Research graph, then run a second ET calculation with those added as trusted nodes to your set of directly trusted nodes`. Both of these compositions are good to have as available options.
> 
> Can you expand on this a little bit? And if it's different also touch on a secondary flavor?

Sure! The first type of composition works like:

1.  Calculate the base graph (`AGI Safety Research`)
2.  Use the results times outflow of flavor `AGI Safety Ecosystem Development` to determine a node's score

This is the same style of composition as flagging.

The second type goes like:
1. Calculate the base graph (`AGI Safety Research`)
2. Populate a viewers's outflowing karma for `AGI Safety Ecosystem Development` with their personal allocation in addition to the results of the base graph
3. Run an EigenTrust calculation on this graph

**Reply by plexish** [reply]
**General**
> How much you and your network generally trusts a node. The combination of all positive trust graphs.

This one is just taking each other type of positive vote and adding them together, then running ET over it.

**Flag**
> This should be used to notify the system that someone is attempting to manipulate the trust graph, by pressuring, requesting, or encouraging others to vote for them on any subgraph.

Secondary flavor of General.

**Comment by tayler6000** [comment]
@plexish What do you think about having a bounty flavor to see if someone holds true on their bounties?

**Reply by plexish** [reply]
hmmm, might be too sparse data if only on bounty payment? And the receivers would not have a way to collect that flavor generally, so I think not quite the right structure. But general bounty trust (for helping with any type of bounty) might be workable?

**Reply by tayler6000** [reply]
Well yeah, it doesn't just have to be for payment, it's just their trust in the bounty ecosystem, and in theory, if it gets implemented into a bounty platform it would be good.  Maybe on the RB Facebook page (if the flavor gets added) we can make a meta post about how the ETN is available to be used in this way and if people want to they can say "My ETN username is x" in their post for people to verify it.  After MVP we could also make a Facebook bot for tying the ETN with Facebook.

**Reply by plexish** [reply]
trust in the bounty ecosystem seems good, yeah, it can be the main flavor on Superlinear. Integrating with fb might be tricky, as they are quite opposed to scraping and many forms of bots. Worth looking into though, might be more practical than I imagine
